### ISSUE TITLE
 Run meta-analysis without lab-level exclusion of n >= 10, report in supplementals

### ISSUE THREAD

**Issue by kristabh** [manybabies/mb1b-analysis-public#1]
p 29: "Note that unlike for the meta-analysis..." --> This isn’t a change from the original plan, but the exclusion for the meta-analysis is. I think you should do the analysis as specified in the Stage 1 document and also do the the meta-analysis with exclusions as an follow-up, exploratory robustness analysis (with clear documentation that it was not planned that way).

**Comment by angelinetsui** [comment]
@kristabh , do you think that we still need a bit of exclusion criteria here. For example, some labs only had 1 infant per language group.....

Otherwise, I think I can re-run the meta-analysis (with certain exclusion criteria, e.g., n >=5) and report it in supplementals

**Comment by kristabh** [comment]
I had forgotten that some labs had 1 infant per language group - hw many cases were there like this?  Perhaps we can just point this out, say that it was therefore impossible to run the analysis without some kind of lab-level inclusion criterion, and argue for why 10 was a reasonable minimum.  It seems weird to just keep running with different arbitrary criteria, although we could.

**Comment by angelinetsui** [comment]
Copied and pasted part of my script in 04.Rmd here:

For matched dataset in the meta-analytical approach: 
1) each lab has at least 10 babies (n>=10) for each age group in both monolingual & bilingual groups;
2)The following bilingual labs are removed based on critieria (1): 
- babylabkingswood 6-9 mo: n=9 in bilingual 6-9 mo gp
- babylabparisdescartes1 all data: no monolinguals at 6-9 mo and n=1 in bilingual 12-15 mo gp
- babylabprinceton all data: n=1 in monolingual 6-9 mo gp
- bllumanitoba all data: n=7 in bilingual 6-9 mo gp and n=8 in bilingual 12-15 mo gp
- langlabucla all data: n=9 in bilingusl 12-15 mo gp and n=3 in monolingual 12-15 mo gp 
- ldlottawa 6-9 mo: n=7 in bilingual
- lllliv all data: n=7 in bilingual 6-9 mo and n=6 in bilingual 12-15 mo
- wsigoettingen all data: n=9 in bilingual 6-9 mo gp and n=7 in bilingual 12-15 mo gp

In short, there were two labs that had one infant in a language group!

**Comment by kristabh** [comment]
Okay, let's run it with a minimum of n = 5 for robustness, report that in the supplemental. I've added some wording both in the response and in the manuscript why these kind of exclusions were unavoidable (i.e., can't compute an effect size from n = 1). See if you can improve the wording.

**Comment by angelinetsui** [comment]
looks good, will do!

**Comment by angelinetsui** [comment]
Re-ran all the meta-analyses using n>=5 criteria. All results are the same, the effect sizes are very similar, I wonder if we can refer readers to the GitHub codes later. I really do not think that it is worthwhile to put everything in the supplemental materials, as it makes the paper unnecessarily long

**Comment by angelinetsui** [comment]
I will close this issue as I have updated the codes on Github.
